client/client.go

```diff
@@ -162,16 +162,11 @@ type (
 	// StartWorkflowOptions configuration parameters for starting a workflow execution.
 	StartWorkflowOptions = internal.StartWorkflowOptions
 
-	// WithStartWorkflowOperation is a type of operation that can be executed as part of a workflow start.
-	// For example, use NewUpdateWithStartWorkflowOperation to perform Update-with-Start.
+	// WithStartWorkflowOperation defines how to start a workflow when using Update-With-Start.
+	// See NewWithStartWorkflowOperation for details.
```

Comment:
```suggestion
	// See [Client.NewWithStartWorkflowOperation] for details.
```

I think this will make it a link

Reply:
Done